var/spack/repos/builtin/packages/mercury/package.py

```diff
@@ -15,18 +15,19 @@ class Mercury(CMakePackage):
     maintainers = ['soumagne']
     tags = ['e4s']
     version('master', branch='master', submodules=True)
+    version('2.1.0', sha256='9a58437161e9273b1b1c484d2f1a477a89eea9afe84575415025d47656f3761b')
     version('2.0.1', sha256='335946d9620ac669643ffd9861a5fb3ee486834bab674b7779eaac9d6662e3fa')
     version('2.0.0', sha256='9e80923712e25df56014309df70660e828dbeabbe5fcc82ee024bcc86e7eb6b7')
     version('1.0.1', sha256='02febd56c401ef7afa250caf28d012b37dee842bfde7ee16fcd2f741b9cf25b3')
     version('1.0.0', sha256='fb0e44d13f4652f53e21040435f91d452bc2b629b6e98dcf5292cd0bece899d4')
     version('0.9.0', sha256='40868e141cac035213fe79400f8926823fb1f5a0651fd7027cbe162b063843ef')
 
     variant('bmi', default=False, description='Use BMI plugin')
-    variant('cci', default=False, description='Use CCI plugin')
     variant('mpi', default=False, description='Use MPI plugin')
     variant('ofi', default=True,  description='Use OFI libfabric plugin')
     # NOTE: the sm plugin does not require any package dependency.
     variant('sm',  default=True,  description='Use shared-memory plugin')
+    variant('ucx', default=False, description='Use UCX plugin')
```

Comment:
We have conditional variants since last release :slightly_smiling_face: 
```suggestion
    variant('ucx', default=False, when='@2.1.0:', description='Use UCX plugin')
```